docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -6298,6 +6298,182 @@ ecl zapgen W20171018-091399 --path ~ --inc-thor-slave-logs --description "Unexpe
             </tgroup>
           </informaltable></para>
       </sect2>
+
+      <sect2 id="CT-RCTL-MEMLOCK" role="brk">
+        <title>memlock</title>
```

Comment:
the title should be:
<title>ecl roxie memlock</title>
(see the other commands in the book for consistency)